public/components/admin.js

```diff
@@ -194,7 +195,7 @@ class CardCategoryDocuments extends React.Component {
       let isRequired = (doc.required) ? 'required' : 'not required';
       docs.push(
         <li data-name={doc.name} key={i}>
-          <span className="card-category-document-name">{doc.name}</span>
+          <span id={doc.name} className="card-category-document-name">{doc.name}</span>
```

Comment:
Why an `id`? There could be many documents with the same name for all different categories (there are already). Could be changed to `categoryname-docname` (but you need to pass down the category name if it's not there).